.github/workflows/main.yml

```diff
@@ -16,11 +15,36 @@ jobs:
       - name: Checkout Repo
         uses: actions/checkout@v3
 
+      - name: Read Review Guidelines
+        id: read-guidelines
+        run: |
+          REVIEW_GUIDELINES=$(cat REVIEW_GUIDE.md)
+          echo "::set-output name=review_guidelines::$REVIEW_GUIDELINES"
+
       - name: AI Code Review Action
         uses: freeedcom/ai-codereviewer@v2.6.0
         with:
           GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
           OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
           OPENAI_API_MODEL: "gpt-4" # Optional: defaults to "gpt-4"
-          exclude: "**/*.json, **/*.md" # Optional: exclude patterns separated by commas
+          exclude: "**/*.json,**/*.md" # Optional: exclude patterns separated by commas
+          review-guidelines: ${{ steps.read-guidelines.outputs.review_guidelines }}
           post-if-error: 'true'
+# on:
```

Comment:
It seems like you have commented out the `on` block which specifies when the workflow should be triggered. If you want this workflow to run on pull requests, you should uncomment this block.